docs/developing.md

```diff
@@ -46,8 +46,8 @@ The Sandstorm ecosystem is full of people who want to promote your app, give you
 it.
 
 - **Getting help**: [Community feedback and Q&A](https://groups.google.com/d/forum/sandstorm-dev) | [Real-time IRC chat on libera.chat](https://kiwiirc.com/client/irc.libera.chat/?channel=#sandstorm) | [Watch presentations on the Sandstorm YouTube channel](https://www.youtube.com/channel/UC8xKZRW86Fa9W00uAppBXXg)
-- **Publicity**: [Give a meetup/conference talk about your app](https://sandstorm.io/news/2015-12-17-community-talks) | [Public demo service for all Sandstorm apps](https://sandstorm.io/news/2015-02-06-app-demo)
-- **Read more**: [All community resources](https://sandstorm.io/community)
+- **Publicity**: [Share news about a meetup/conference talk about your app](https://sandstorm.org/news/) | [Public demo service for all Sandstorm apps](https://sandstorm.io/news/2015-02-06-app-demo)
```

Comment:
I am not sure I would just point to /news here. And there's a second .io link on this line. Note that the .org site does include all the original blog posts, so you could just update the .io to .org for both.